javascripts/discourse/templates/components/dc-category-card.hbs

```diff
@@ -20,5 +20,13 @@
     <p class="m-0 ml-1 dc-text-light">
       {{category.totalTopicCount}}
     </p>
+    {{#if category.tdc_is_collective}}
```

Comment:
@duranmla what about moving this check to a method, the `groupCount` method is doing the same check so feels this can be just one method

Reply:
I guess we can simply say:

```
{{#if groupCount}}
    ...
{{/if}}
```
The _"problem"_ is that `0` is going to evaluate `false` in that sense, all collectives without members are not going to show the icon [like this](https://user-images.githubusercontent.com/1425162/80390616-f71f5680-88ac-11ea-856b-c49d21de67c9.png). But I guess that won't be a use case? Let me know what do you think.

Reply:
I think it's fine, if it's going to add complexity let's leave it like that